content/en/docs/ops/best-practices/security/index.md

```diff
@@ -7,36 +7,170 @@ owner: istio/wg-security-maintainers
-The photo SRE team creates two service accounts to run `photo-frontend` and
-`photo-backend` respectively in the `photo-ns` namespace. The datastore SRE
-team creates one service account to run the `datastore` service in the
-`datastore-ns` namespace. Moreover, we need to enforce the service access
-control in [Istio Mixer](https://istio.io/v1.6/docs/reference/config/policy-and-telemetry/) such that
-`photo-frontend` cannot access datastore.
-
-In this setup, Kubernetes can isolate the operator privileges on managing the services.
-Istio manages certificates and keys in all namespaces
-and enforces different access control rules to the services.
+Istio security features provide strong identity, powerful policy, transparent TLS encryption, and authentication, authorization and audit (AAA) tools to protect your services and data.
+However, to fully make use of these features securely, care must be taken to follow best practices. It is recommended to review the [Security overview](/docs/concepts/security/) before proceeding.
+
+## Mutual TLS
+
+Istio will [automatically](/docs/ops/configuration/traffic-management/tls-configuration/#auto-mtls) encrypt traffic using [Mutual TLS](/docs/concepts/security/#mutual-tls-authentication) whenever possible.
+However, proxies are configured in [permissive mode](/docs/concepts/security/#permissive-mode) by default, meaning they will accept both mutual TLS and plaintext traffic.
+
+While this is required for incremental adoption or allowing traffic from clients without an Istio sidecar, it also weakens the security stance.
+It is recommended to [migrate to strict mode](/docs/tasks/security/authentication/mtls-migration/) when possible, to enforce that mutual TLS is used.
+
+Mutual TLS alone is not always enough to fully secure traffic, however, as it provides only authentication, not authorization.
+This means that anyone with a valid certificate can still access a service.
+
+To fully lock down traffic, it is recommended to configure [authorization policies](/docs/tasks/security/authorization/).
+These allow creating fine-grained policies to allow or deny traffic. For example, you can allow only requests from the `app` namespace to access the `hello-world` service.
+
+## Understand traffic capture limitations
+
+The Istio sidecar works by capturing both inbound traffic and outbound traffic and directing them through the sidecar proxy.
+
+However, not *all* traffic is captured:
+
+* Redirection only handles TCP based traffic. Any UDP or ICMP packets will not be captured or modified.
+* Inbound capture is disabled on many [ports used by the sidecar](/docs/ops/deployment/requirements/#ports-used-by-istio) as well as port 22. This list can be expanded by options like `traffic.sidecar.istio.io/excludeInboundPorts`.
+* Outbound capture may similarly be reduced through settings like `traffic.sidecar.istio.io/excludeOutboundPorts` or other means.
```

Comment:
I wonder if there should be a stronger link between this section and the immediately following NetworkPolicy section?   You could explain which of the issues raised here can be addressed with k8s NetworkPolicy